### DISCUSSION TITLE
Add a beatmap text editor

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#25965] [Ideas] [answered]
In the stable version of osu!, in the beatmap editor you can open the `.osu` file (and the storyboard file) in a text editor.
Being able to do the same or including a text editor within Lazer would be a nice addition.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/12029

**Reply by yesseruser** [reply]
> https://github.com/ppy/osu/issues/12029

Thanks.
